spec/rubocop/rspec/example_spec.rb

```diff
@@ -9,12 +9,12 @@ def example(source)
 
   it 'extracts doc string' do
     expect(example("it('does x') { foo }").doc_string)
-      .to eql(s(:str, 'does x'))
+      .to eql(RuboCop::AST::StrNode.new(:str, ['does x']))
   end
 
   it 'extracts doc string for unimplemented examples' do
     expect(example("it('does x')").doc_string)
-      .to eql(s(:str, 'does x'))
+      .to eql(RuboCop::AST::StrNode.new(:str, ['does x']))
```

Comment:
@pocke Does this look right? It is a bit confusing that I can’t tell the difference between `Node#inspect` and `StrNode#inspect`:
```
    Failure/Error:
       expect(example("it('does x')").doc_string)
         .to eql(s(:str, 'does x'))

       expected: s(:str, "does x")
            got: s(:str, "does x")
```

Reply:
`AST::Node#eql?` checks classes. https://github.com/whitequark/ast/blob/241cac1bea04fd7dfd9a83959713d7fa12b4e2ec/lib/ast/node.rb#L85-L89

Maybe the test cases does not expect the class equivalent. So I think we can replace `.to eql` with `.to eq`, and use `s` instead of `RuboCop::AST::StrNode.new`.

Reply:
Thank you – I’ll switch to using `eq` instead.